exercises/ruby-1-b/.meta/config.json

```diff
@@ -3,39 +3,72 @@
   "test_file": "ruby_1_b_test.rb",
   "tests": [
     {
```

Comment:
We've recently added the `"expected"` property to each test entry. See https://github.com/exercism/research_experiment_1/blob/master/exercises/ruby-1-a/.meta/config.json#L9 Could you add the `"expected"` value to all entries? FYI: this is how expected `string` values should be encoded: https://github.com/exercism/research_experiment_1/blob/master/exercises/ruby-1-a/.meta/config.json#L27

Reply:
I added the key for most of the tests, but I wasn't sure how I should represent return values which are objects. Should I maybe change the methods to return simpler data types like a boolean? e.g. there's no real reason why `borrow` couldn't return `true` instead of a book object